src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -30,18 +33,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```

Comment:
Thrown, like it should be. Good, but exception message could be more descriptive

Reply:
Im on it!